ansible/roles/kong-api/defaults/main.yml

```diff
@@ -349,7 +349,25 @@ kong_apis:
         - 'contentTempAccess'
```

Comment:
@shiva-rakshith Please remove this ACL if mobile apps do not need access to this API

Reply:
For the existing api, we are not sure. But for the new api mobile apps do not need access.

Reply:
Ok, then we need to change this if you are unsure for now

Reply:
Can we also revert the config.hour of existing api to 'premium_consumer_large_rate_limit_per_hour', we are not unsure if it disturbs anything.

Reply:
No, we will not revert that. We do not want to use premimu rate limits unless we are 100% sure about it.

Reply:
@maheshkumargangula - For new PrivateSearch API, we are using config.hour as 'medium_rate_limit_per_hour'. @keshavprasadms has suggested to use the same config.hour for existing search API. Can we change from 'premium_consumer_large_rate_limit_per_hour' to 'medium_rate_limit_per_hour' for existing search API. Please confirm.